pom.xml

```diff
@@ -1,192 +1,192 @@
-            <id>make-assembly</id>
-            <!-- this is used for inheritance merges -->
-            <phase>package</phase>
-            <!-- bind to the packaging phase -->
-            <goals>
-              <goal>single</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
-        <artifactId>maven-dependency-plugin</artifactId>
-        <executions>
-          <execution>
-            <phase>initialize</phase>
-            <goals>
-              <goal>copy-dependencies</goal>
-            </goals>
-            <configuration>
-              <overWriteReleases>false</overWriteReleases>
-              <includeScope>runtime</includeScope>
-              <outputDirectory>${project.build.directory}/lib</outputDirectory>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-jar-plugin</artifactId>
-        <version>3.1.0</version>
-        <configuration>
-          <archive>
-            <manifest>
-              <addClasspath>true</addClasspath>
-              <classpathPrefix>lib/</classpathPrefix>
-              <mainClass>com.kpn.dsh.example.Main</mainClass>
-            </manifest>
-          </archive>
-        </configuration>
-      </plugin>
-      <plugin>
-        <groupId>com.spotify</groupId>
-        <artifactId>dockerfile-maven-plugin</artifactId>
-        <version>1.3.3</version>
-        <executions>
-          <execution>
-            <id>default</id>
-            <goals>
-              <goal>build</goal>
-              <goal>push</goal>
-            </goals>
-          </execution>
-        </executions>
-        <configuration>
-          <repository>dataserviceshub-docker-${tenant}.jfrog.io/tenant-example</repository>
-          <tag>${project.version}</tag>
-          <buildArgs>
-            <IMAGE_VERSION>${project.version}</IMAGE_VERSION>
-          </buildArgs>
-        </configuration>
-      </plugin>
-    </plugins>
-  </build>
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.kpn-dsh</groupId>
+    <artifactId>tenant-example</artifactId>
+    <version>1.0.1-SNAPSHOT</version>
+    <properties>
+        <tenant>dshdemo1</tenant>
+        <os-maven-plugin.version>1.5.0.Final</os-maven-plugin.version>
+        <protobuf.version>3.21.9</protobuf.version>
+        <org.slf4j.version>2.0.4</org.slf4j.version>
+        <ch.qos.logback.version>1.4.5</ch.qos.logback.version>
+    </properties>
+    <dependencies>
+        <!-- Protobuf -->
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+            <version>${protobuf.version}</version>
+        </dependency>
+        <!-- Kafka -->
+        <dependency>
+            <groupId>org.apache.kafka</groupId>
+            <artifactId>kafka-clients</artifactId>
+            <version>3.3.1</version>
```

Comment:
We don't use Kafka 3 yet. We recommend https://search.maven.org/artifact/org.apache.kafka/kafka-clients version 2.7.2 at the moment. You can try the newer 2.8.2 also. But I would stay away from 3.x for now.